examples/bri-3/src/app.module.ts

```diff
@@ -9,7 +9,7 @@ import { WorkgroupsModule } from './bri/workgroup/workgroup.module';
 import { LoggingModule } from './shared/logging/logging.module';
 import { AuthModule } from './bri/auth/auth.module';
 import { APP_GUARD } from '@nestjs/core';
-import { JwtAuthGuard } from './bri/auth/guards/sri-jwt.guard';
```

Comment:
renamed the file to not use sri- prefix since that is not convention for naming filess